pytest_splunk_addon/standard_lib/data_models/Change.json

```diff
@@ -1,217 +1,376 @@
+                },
+                {
+                    "name": "vendor_account",
+                    "type": "optional",
+                    "comment": "The account that manages the user that initiated the request. The account represents the organization, or a Cloud customer or a Cloud account."
+                },
+                {
+                    "name": "vendor_region",
+                    "type": "optional",
+                    "comment": "The account that manages the user that initiated the request. The account represents the organization, or a Cloud customer or a Cloud account."
+                },
+                {
+                    "name": "user_type",
+                    "type": "optional",
+                    "comment": "The type of the user involved in the event or who initiated the event, such as IAMUser, Admin, or System. For account management events, this should represent the type of the user changed by the request.."
+                },
+                {
+                    "name": "vendor_product",
+                    "type": "required",
+                    "comment": "The vendor and product or service that detected the change. This field can be automatically populated by vendor and product fields in your data."
+                }
+            ],
+            "child_dataset": [
+                {
+                    "name": "Account_Management",
+                    "tags": [
+                        [
+                            "change",
+                            "account"
+                        ]
+                    ],
+                    "fields_cluster": [],
+                    "search_constraints": "tag=account",
+                    "fields": [
+                        {
+                            "name": "dest_nt_domain",
+                            "type": "optional",
+                            "comment": "The NT domain of the destination, if applicable."
+                        },
+                        {
+                            "name": "src_nt_domain",
+                            "type": "optional",
+                            "comment": "The NT domain of the source, if applicable."
+                        },
+                        {
+                            "name": "src_user",
+                            "type": "conditional",
+                            "condition": "object_category=user",
+                            "comment": "For account changes, the user or entity performing the change."
+                        },
+                        {
+                            "name": "src_user_name",
+                            "type": "optional",
+                            "comment": "For account changes, the user name of the user or entity performing the change."
+                        }
+                    ],
+                    "child_dataset": [
```

Comment:
Why do we have child datasets from `Account_Management`? It is not in the official CIM documenation.

Reply:
So these data sets were already present and it is not a diff between v4.15 vs 5.3.2.

Reply:
Hmm, I see the Account_Management dataset is in the official CIM docs and it is in the official CIM [.json](https://cd.splunkdev.com/EnterpriseSecurity/sa-commoninformationmodel/-/blob/develop/packages/sa-commoninformationmodel/src/main/resources/splunk/default/data/models/Certificates.json#L206)